### PR TITLE
Iss

### PR DESCRIPTION
**Pull request by lolaolah**
space station location pull request!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-node_modules
```
Comment (ziad-saab) [inline .gitignore]: You did not have to delete this, only look at it and notice what effect it had. To make sure, can you please explain to me in your own words what this does?

**Comment by ziad-saab** [comment]
This is a great submission! Please answer the question about `.gitignore` and I will close it :)

File: iss.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var request = require("request");
+var url = 'http://api.open-notify.org/iss-now.json';
+
+request(url, function (error, response, body) {
+  if (!error && response.statusCode == 200) {
+    var fbResponse = JSON.parse(body);
+    
+    console.log(fbResponse);
```
Comment (nyanofthemoon) [inline iss.js]: Only output the latitude and the longitude.

File: .gitignore (hunk 1 above)
Comment (lolaolah) [inline .gitignore]: when you add a file in a .gitignore file, GIT ignores the file when it makes a commit.

Comment (ziad-saab) [inline .gitignore]: yes :)

**Comment by ziad-saab** [comment]
:+1:

**Pull request closed without merging**